src/main/java/uk/gov/hmcts/darts/task/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -254,6 +281,24 @@ private void rescheduleCloseNonCompletedTranscriptionsAutomatedTask() {
         }
     }
 
+    private void rescheduleOutboundAudioDeleterAutomatedTask() {
+        OutboundAudioDeleterAutomatedTask outboundAudioDeleterAutomatedTask;
+        Trigger trigger;
```

Comment:
These don't need to be defined outside there usage scope, I see its been copied from above. I would change those too.

Reply:
ACCEPT